diff --git a/node_modules/wappalyzer/driver.js b/node_modules/wappalyzer/driver.js
index abfbcad..768754b 100644
--- a/node_modules/wappalyzer/driver.js
+++ b/node_modules/wappalyzer/driver.js
@@ -15,8 +15,8 @@ const { CHROMIUM_BIN, CHROMIUM_DATA_DIR, CHROMIUM_WEBSOCKET, CHROMIUM_ARGS } =
 const chromiumArgs = CHROMIUM_ARGS
   ? CHROMIUM_ARGS.split(' ')
   : [
-      '--headless',
-      '--single-process',
+      '--headless=old', // We upgraded puppeteer but still we want to use the old headless mode since Wappalyzer has been only developed/tested for it
+      // '--single-process',
       '--no-sandbox',
       '--no-zygote',
       '--disable-gpu',
@@ -543,7 +543,7 @@ class Site {
     errorMessage = 'Operation took too long to complete',
     maxWait = this.options.fast
       ? Math.min(this.options.maxWait, 2000)
-      : this.options.maxWait
+      : (4 * this.options.maxWait) // Increasing the default timeout since async processes in JavaScript will take longer than usual since we use concurrency (the real `maxWait` is only meaningful when talking to Puppeteer)
   ) {
     let timeout = null
 
